src/eventProcessor.ts

```diff
@@ -360,26 +409,34 @@ export default async (event: ContractEvent): Promise<void> => {
 
       const tokenAddress = await getColonyTokenAddress(colonyAddress);
 
-      verbose(
-        amount.toString(),
-        'tokens were minted in colony:',
-        colonyAddress,
-      );
+      if (amount && amount.toString() !== '0') {
```

Comment:
Is the motion set to very short timeout ? Ie: it finishes instantly ? Otherwise I can't really explain this.

The underlying action / event doesn't get triggered until the motion finishes (and that is done manually).

If that isn't the case I don't really understand what's going on. Maybe ask @area for his input ?